ddclient.in

```diff
@@ -2524,6 +2538,10 @@ sub check_value {
         $value = lc $value;
         return undef if !exists $protocols{$value};
 
+    } elsif ($type eq T_URL) {
+        $value = lc $value;
+        return undef if $value !~ /[^.\/]+(\.[^.\/]+)+(:\d+)?(\/[^\/]+)*\/?$/;
```

Comment:
Wasn't sure how strict to make this regex, happy to relax this if that would be more in line with the project.